apps/assessment_registry/models.py

```diff
@@ -0,0 +1,119 @@
+from django.db import models
+from apps.user_resource.models import UserResource
+from apps.geo.models import Region
+from apps.organization.models import Organization
+# Create your models here.
+
+
+class Assessment_Registry(UserResource):
+    class CrisisType(models.IntegerChoice):
+        EARTH_QUAKE = 1, 'Earth Quake'
+        GROUND_SHAKING = 2, 'Ground Shaking'
+        TSUNAMI = 3, 'Tsunami'
+        VOLCANO = 4, 'Volcano'
+        VOLCANIC_ERUPTION = 5, 'Volcanic Eruption'
+        MASS_MOMENT_DRY = 6, 'Mass Movement (Dry)'
+        ROCK_FALL = 7, 'Rockfall'
+        AVALANCE = 8, 'Avalance'
+        LANDSLIDE = 9, 'Landslide'
+        SUBSIDENCE = 10, 'Subsidence'
+        EXTRA_TROPICAL_CYCLONE = 11, 'Extra Tropical Cyclone'
+        TROPICAL_CYCLONE = 12, 'Tropical Cyclone'
+        LOCAL_STROM = 13, 'Local/Convective Strom'
+        EXTRA_TROPICAL_CYCLONE = 14, 'Extra Tropical Cyclone'
+        FLOOD_RAIN = 15, 'Flood/Rain'
+        GENERAL_RIVER_FLOOD = 16, 'General River Flood'
+        FLASH_FLOOD = 17, 'Flash flood'
+        STROM_SURGE_FLOOD = 18, 'Strom Surge/Coastal Flood'
+        MASS_MOVEMENT_WET = 19, 'Mass Movement (Wet)'
+        EXTREME_TEMPERATURE = 20, 'Extreme Temperature'
+        HEAT_WAVE = 21, 'Heat Wave'
+        COLD_WAVE = 22, 'Cold Wave'
+        EXTREME_WEATHER_CONDITION = 23, 'Extreme Weather Condition'
+        DROUGHT = 24, 'Drought'
+        WILDFIRE = 25, 'Wildfire'
+        POPULATION_DISPLACEMENT = 112, 'Population Displacement'
+        CONFLICT = 79, 'Conflict'
```

Comment:
Why are the values so different?

Reply:
To avoid adding a separate data migration mapping from the existing one to the new one, we are using the existing value instead.
Maybe adding a NOTE should do [review-comment](https://github.com/the-deep/server/pull/1290/files/54606caeaef57a0d6c9bd9f6315094cea8e3f01d#diff-12c84566837f69a7fae6f07afd3bb42dee0995419d2fb20921adef287fb3d70a)